src/storages/interfaces.py

```
@@ -0,0 +1,26 @@
+class S3StorageInterface(ABC):
+
+    @abstractmethod
+    async def upload_file(self, file_name: str, file_data: Union[bytes, bytearray]) -> None:
+        """
+        Uploads a file to the storage.
+
+        :param file_name: The name of the file to be stored.
+        :param file_data: The file data in bytes.
+        :return: URL of the uploaded file.
+        """
+        pass
```

Pylint (3.3.7)

[warning] Unnecessary pass statement

(W0107)